regression/cbmc-java/lazyloading_indirect_generic_array_parameter/test.desc

```diff
@@ -0,0 +1,7 @@
+KNOWNBUG
```

Comment:
Could you create and link to to a jira ticket to avoid this test getting lost and then duplicated.